packages/lldb/build.sh

```diff
@@ -1,39 +1,45 @@
 TERMUX_PKG_HOMEPAGE=https://lldb.llvm.org
 TERMUX_PKG_DESCRIPTION="LLVM based debugger"
 TERMUX_PKG_LICENSE="NCSA"
-TERMUX_PKG_VERSION=9.0.1
-TERMUX_PKG_SRCURL=https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/lldb-$TERMUX_PKG_VERSION.src.tar.xz
-TERMUX_PKG_SHA256=8a7b9fd795c31a3e3cba6ce1377a2ae5c67376d92888702ce27e26f0971beb09
+TERMUX_PKG_VERSION=10.0.0
+TERMUX_PKG_SRCURL=(https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/lldb-$TERMUX_PKG_VERSION.src.tar.xz
+		   https://github.com/llvm/llvm-project/releases/download/llvmorg-$TERMUX_PKG_VERSION/llvm-$TERMUX_PKG_VERSION.src.tar.xz)
+TERMUX_PKG_SHA256=(dd1ffcb42ed033f5167089ec4c6ebe84fbca1db4a9eaebf5c614af09d89eb135
+		   df83a44b3a9a71029049ec101fb0077ecbbdf5fe41e395215025779099a98fdf)
 TERMUX_PKG_DEPENDS="libc++, libedit, libllvm, libxml2, ncurses-ui-libs"
 TERMUX_PKG_BUILD_DEPENDS="libllvm-static"
 TERMUX_PKG_BREAKS="lldb-dev, lldb-static"
 TERMUX_PKG_REPLACES="lldb-dev, lldb-static"
+TERMUX_PKG_HOSTBUILD=true
 TERMUX_PKG_HAS_DEBUG=false
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
--DLLDB_DISABLE_CURSES=0
--DLLDB_DISABLE_LIBEDIT=0
--DLLDB_DISABLE_PYTHON=1
--DLLVM_CONFIG=$TERMUX_PREFIX/bin/llvm-config
+-DLLDB_TABLEGEN=$TERMUX_PKG_HOSTBUILD_DIR/bin/lldb-tblgen
+-DLLVM_ENABLE_SPHINX=ON
 -DLLVM_ENABLE_TERMINFO=1
 -DLLVM_LINK_LLVM_DYLIB=ON
 -DLLVM_DIR=$TERMUX_PREFIX/lib/cmake/llvm
--DClang_DIR=$TERMUX_PREFIX/lib/cmake/clang
--DLLVM_NATIVE_BUILD=$TERMUX_PREFIX/bin
+-DLLVM_TABLEGEN=$TERMUX_PKG_HOSTBUILD_DIR/llvm/bin/llvm-tblgen
```

Comment:
The first three `DISABLE_` flags are ignored just like `LLVM_CONFIG`, it figures out `Clang_DIR` based on the `LLVM_DIR`, and specifying `LLVM_TABLEGEN` replaces `LLVM_NATIVE_BUILD`.